### ISSUE TITLE
CC Buchner download error

### ISSUE THREAD

**Issue by pkoe1975** [RythenGlyth/EbookDownloader#22]
> ebookdownloader@1.0.0 start
> node --no-deprecation src/EbookDownloader.js

√ Publisher / Website » C.C.BUCHNER - click & study
√ Email (Empty to read from config.json) ... my.email.at.home@gmail.com
√ Password (Empty to read from config.json) ... ********
√ Overwrite old temp files ... yes
√ Select a book » Mathe.Logo Bayern 9 I – neu
√ Selectable text ... off / on
deleted temp files
created folder: ./out/DownloadTemp/Mathe.Logo Bayern 9 I  neu/
downloaded 0/310 pages
node:events:497
      throw er; // Unhandled 'error' event
      ^

Error: spawn ffmpeg ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:285:19)
    at onErrorNT (node:internal/child_process:483:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess._handle.onexit (node:internal/child_process:291:12)
    at onErrorNT (node:internal/child_process:483:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn ffmpeg',
  path: 'ffmpeg',
  spawnargs: [ '-f', 'jpeg_pipe', '-i', '-', '-f', 'image2', '-' ]
}

Node.js v22.9.0

I love using this tool, and my other downloads (cornelsen, klett) work like a breeze. only cc buchner always shows this error....  Any idea appreciated.
Thanks, Pete

**Comment by RythenGlyth** [comment]
If you want you can send me your credentials via email ([my name]@gmail.com) and I can look into it.

**Comment by TudbuT** [comment]
have you installed ffmpeg?

**Comment by pkoe1975** [comment]
Yes, but thank you for the hint! It was rather old, so I updated ffmpeg to new version and it works now!  Thank you all and keep up the good work!
